kvdb-rocksdb/src/lib.rs

```diff
@@ -443,6 +443,18 @@ impl Database {
 							stats_total_bytes += key.len();
 							batch.delete_cf(cf, &key).map_err(other_io_err)?
 						}
+						DBOp::DeletePrefix { col: _, prefix } => {
+							if prefix.len() > 0 {
+								let end_range = kvdb::end_prefix(&prefix[..]);
+								batch.delete_range_cf(cf, &prefix[..], &end_range[..]).map_err(other_io_err)?;
+							} else {
+								// delete every values in the column
+								let end_range = &[255u8];
+								batch.delete_range_cf(cf, &prefix[..], &end_range[..]).map_err(other_io_err)?;
+								batch.delete_cf(cf, &end_range[..]).map_err(other_io_err)?;
+								batch.delete_cf(cf, &[]).map_err(other_io_err)?;
```

Comment:
is this redundant?

Reply:
Oh yes, start is [] so it should already be in range.